test/module/irohad/consensus/yac/mocks/crypto_provider.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gmock/gmock.h>
```

Comment:
Includes should be inside header guards
https://stackoverflow.com/a/29954513
Please check other files too